metadata/v1.4/enable_ed25519_pubkey_validate_return_false_wrong_length.json

```diff
@@ -1,6 +1,6 @@
 {
   "title": "Enable ed25519_pubkey_validate_return_false_wrong_length",
-  "description": "AIP-23: Make ed25519 public key validation return none if key has the wrong length.",
-  "source_code_url": "TBD",
+  "description": "AIP-23: Changes the function native_public_key_validate used by the native Move function native fun pubkey_validate_internal to return false if the public key provided is the wrong length. Previously, this function would abort if the key length provided was incorrect.",
+  "source_code_url": "https://github.com/aptos-foundation/mainnet-proposals/blob/main/sources/v1.4/enable_ed25519_pubkey_validate_return_false_wrong_length/0-features.move",
```

Comment:
I think pointing to the folder should be enough, just in case for certain proposal there's multiple source code files. like this `https://github.com/aptos-foundation/mainnet-proposals/blob/main/sources/v1.4/enable_ed25519_pubkey_validate_return_false_wrong_length`

Reply:
updated!